spec/units/services/unsubscribe_service_spec.rb

```diff
@@ -1,6 +1,6 @@
 RSpec.describe UnsubscribeService do
   describe ".subscriber!" do
-    let!(:subscriber) { FactoryGirl.create(:subscriber, address: "foo@bar.com") }
+    let!(:subscriber) { FactoryBot.create(:subscriber, address: "foo@bar.com") }
```

Comment:
We could just use `create` here too.

Reply:
Dang - I missed that one. I'm very sloppy today! Thanks 👍